back-end/slothstagram/urls.py

```diff
@@ -17,14 +17,13 @@
 from django.urls import path, include
 
 from rest_framework import routers
-from photo_app.views import PhotoViewSet, UserViewSet
 
 router = routers.DefaultRouter()
-router.register(f'photo', PhotoViewSet)
-router.register(f'user', UserViewSet)
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('photo_app.urls')),
+    path('', include(router.urls)),
+
```

Comment:
This line is unnecessary since `router.urls` is empty.
Delete line 26 and line 22.